frontend/src/metabase/static-viz/components/CategoricalLineChart/CategoricalLineChart.jsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { AxisBottom, AxisLeft } from "@visx/axis";
+import { GridRows } from "@visx/grid";
+import { scaleBand, scaleLinear } from "@visx/scale";
+import { LinePath } from "@visx/shape";
+import { Text } from "@visx/text";
+import {
+  getXTickWidth,
+  getXTickLabelProps,
+  getYTickLabelProps,
+  getYTickWidth,
+  getXTickHeight,
+} from "../../lib/axes";
+import { formatNumber } from "../../lib/numbers";
+import { truncateText } from "../../lib/text";
+
+const propTypes = {
+  data: PropTypes.array.isRequired,
+  accessors: PropTypes.shape({
+    x: PropTypes.func.isRequired,
+    y: PropTypes.func.isRequired,
+  }).isRequired,
+  settings: PropTypes.shape({
+    x: PropTypes.object,
+    y: PropTypes.object,
+  }),
+  labels: PropTypes.shape({
+    left: PropTypes.string,
+    bottom: PropTypes.string,
+  }),
+};
+
+const layout = {
+  width: 540,
+  height: 300,
+  margin: {
+    top: 0,
+    left: 55,
+    right: 40,
+    bottom: 40,
+  },
+  font: {
+    size: 11,
+    family: "Lato, sans-serif",
+  },
+  colors: {
+    brand: "#509ee3",
```

Comment:
Is this the same "brand" color that we have in the colors lib?

I think those should be imported, rather than hard coded, for the sake of whitelabelling.

Reply:
Nope, static viz code is executed by a standalone JS VM and most of existing codebase cannot be imported, including colors. Currently whitelabelling is not supported.

Reply:
There's a separate issue for whitelabel static viz - issue 18163